src/main/resources/config/liquibase/master.xml

```diff
@@ -33,6 +33,7 @@
     <include file="config/liquibase/changelog/20200420120000_upload_validation_role.xml" relativeToChangelogFile="false"/>
     <include file="config/liquibase/changelog/20200424120000_multiple_validations.xml" relativeToChangelogFile="false"/>
     <include file="config/liquibase/changelog/20200429120000_multiple_validations_foreign_key.xml" relativeToChangelogFile="false"/>
+    <include file="config/liquibase/changelog/20200522120000_increase_validation_feedback.xml" relativeToChangelogFile="false"/>
```

Comment:
please rename the file with the current date and make it the most recent change - otherwise this wouldn't work, because the project_upload_relationship.xml was already applied...